freqtrade/plugins/protections/stoploss_guard.py

```diff
@@ -51,7 +51,6 @@ def _stoploss_guard(self, date_now: datetime, pair: str = None) -> ProtectionRet
         # if pair:
         #     filters.append(Trade.pair == pair)
         # trades = Trade.get_trades(filters).all()
-        # TODO-lev: Liquidation price?
```

Comment:
I believe the liq will just be recorded as a stoploss, so this shouldn't be needed.